src/html5_playback.js

```diff
@@ -119,7 +119,7 @@ export default class HTML5TVsPlayback extends Playback {
     const currentSource = this.$sourceElement && this.$sourceElement.src
     if (sourceURL === currentSource) return
 
-    this.config && this.config.drm && this.config.drm.licenseServerURL && !this._drmConfigured
+    this.config && this.config.drm && this.config.drm && !this._drmConfigured
```

Comment:
The check for `this.config.drm` existance is duplicated here.

Reply:
Btw, the `@babel/preset-env` version used in this project already supports optional chaining. Maybe we could use here to clean it up.